plus/oop/properties.js

```diff
@@ -1,7 +1,18 @@
 /**
  * It returns the property names of the given object.
- * 
+ *
  * @param {object} o the object
  * @returns {string[]} the list of the properties of the object
  *                     or empty array if it is not an object
- */
\ No newline at end of file
+ */
+
+module.exports = function properties(o) {
+  if (typeof o !== "object") {
+    return [];
+  }
+  let props = [];
+  for (let prop in o) {
```

Comment:
Your solution is ok, but we learned about a JS method, which can return with object keys in an array. Do you remember?